src/desktop/package.json

```diff
@@ -117,15 +117,27 @@
     "nsis": {
       "deleteAppDataOnUninstall": true
     },
+    "snap": {
+      "stagePackages": [
+        "default",
+        "libsecret"
+      ],
+      "plugs": [
+        "default",
+        "camera"
```

Comment:
Do we need any of the following?

- [cups-control](https://docs.snapcraft.io/cups-control-interface)
- [password-manager-service](https://docs.snapcraft.io/password-manager-service-interface)
- [raw-usb](https://docs.snapcraft.io/raw-usb-interface)
- [uhid](https://docs.snapcraft.io/uhid-interface)
- [hidraw](https://docs.snapcraft.io/hidraw-interface)

I forget which of the last 3 is used by Ledger so I just posted all of them here